docs/custom-image.md

```diff
@@ -0,0 +1,115 @@
+In this section we will learn how we can create a custom BitOps image with plugins and additional utilities. 
+
+
+
+### Clone Bitops source
+First thing is first, if you haven't already clone the github bitops repo locally
+`git clone https://github.com/bitovi/bitops.git`
+
+<hr/>
+
+
+### Modify the *bitops.config.yaml*
+Using your preferred editor, open the root level `bitops.config.yaml`
+
+This file is used to configure the BitOps image.
+
+[Example bitops.config.yaml](https://github.com/bitovi/bitops/tree/plugins/prebuilt-config)
+
+<hr/>
+
+### bitops.config.yaml
+
+<br/>
+
+#### BitOps "official" image
+Below is an example of how the "official" image of bitops is configured. 
+
+As you can see there are two sections we need to be aware of; Plugins and Deployments. 
+
+<br/>
+
+#### plugins
+Defines a plugin and the source for that plugin. 
+
+<br/>
+
+#### Deployment
+Defines alias', the sequence of executions for those alias' and the alias' related plugin.
+
+
+### official image
+```
+  plugins:    
+    aws:
+      source: https://github.com/bitops-plugins/aws
+    terraform:
+      source: https://github.com/bitops-plugins/terraform
+    cloudformation:
+      source: https://github.com/bitops-plugins/cloudformation
+    helm:
+      source: https://github.com/bitops-plugins/helm
+    kubectl:
+      source: https://github.com/bitops-plugins/kubectl
+    ansible:
+      source: https://github.com/bitops-plugins/ansible
+  deployments:
+    cloudformation:
+      plugin: cloudformation
+    terraform:
+      plugin: terraform
+    helm:
+      plugin: helm
+    ansible:
+      plugin: ansible
+```
+
+<hr/>
+
+### Custom image example
+```
+plugins:
+    plugin-name-1:
+        source: url-to-plugin-1
+    plugin-name-2:
+        source: url-to-plugin-2
+deployments:
+    alias-name-1:
+        plugin: plugin-name-1
+    alias-name-2:
+        plugin: plugin-name-2
+    alias-name-3:
+        plugin: plugin-name-1
+```
+
+In the above example we define 2 plugins that BitOps will install, `plugin-name-1` and `plugin-name-2`. 
```

Comment:
move this sentence above the text block and change the wording to say "in the following example"